internal/services/healthcare/healthcare_dicom_resource.go

```diff
@@ -115,6 +118,86 @@ func resourceHealthcareApisDicomService() *pluginsdk.Resource {
 				Default:  true,
 			},
 
+			"cors_configuration": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"allowed_origins": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeString,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+						},
+						"allowed_headers": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeString,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+						},
+						"allowed_methods": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeString,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+						},
+						"max_age_in_seconds": {
+							Type:     pluginsdk.TypeInt,
+							Optional: true,
```

Comment:
can you validate this? i presume -3 is not a valid value? is 0? is 99999999999 ?

Reply:
Trid it out, can support 0 to 99998 inclusive, added the validatefunc